### DISCUSSION TITLE
Почему не работает перевод по прямой ссылке на mp4 ?

### DISCUSSION THREAD

**Discussion by MyNameIsSoftIce** [ilyhalight/voice-over-translation#633] [Q&A] [answered]
Использую vot-cli . По ссылкам с ютуба всё работает, а по прямой из облака mail.ru что-то не получается.

Строка такая:
vot-cli "https://cloud.mail.ru/public/q7w7/Uk7N33hg1" --output="."

ЧЯДНТ ?

**Comment by NullVerdict** [comment]
> Использую vot-cli . По ссылкам с ютуба всё работает, а по прямой из облака mail.ru что-то не получается.
> 
> Строка такая: vot-cli "https://cloud.mail.ru/public/q7w7/Uk7N33hg1" --output="."
> 
> ЧЯДНТ ?

Ссылка не заканчивается на .mp4

**Comment by ilyhalight** [comment, marked as answer]
На данный момент облако mail.ru не поддерживается, поддерживается только my.mail.ru/video. Прямые ссылки на mp4 - это интернет ссылки (http(s)://), которые заканчиваются на .mp4 (не путать с локальными)

По вопросам связанными с vot-cli в дальнейшем прошу писать в соответствующий [репозиторий](https://github.com/FOSWLY/vot-cli). Дискуссии там включил.
